abc.html

```diff
@@ -0,0 +1,73 @@
+    <nav class="nav-bar">
+        <div><span class="material-symbols-outlined" style="font-size: 48px;color:#272A31;" id="menu-bar">
+            menu
+            </span></div>
+    </nav>
+    <div id="home">
+        <h1 class="top-heading1">"Hello Gamers"</h1>
+        <h2 class="top-heading2">The ti 2022 the greatest of all.</h2>
+        <p class="paragraph">The ti-11 the mega event is approaching the wait is over all of you waiting for.Get ready this NOvermeber the clash between the giant will start there would be 18 teams from the qualifying rounds and there would be only 1 winner who cant it be support your favourite players and teams</p>
+        <h3 id="date">2022.10.15(Wed)-29(Sun)</h3>
+        <p class="location">@ Dubai exclusiverly</p>
+    </div>
+</header>
+<section id="Main-program">
+    <h2 class="main-event">Main Event
+    <div class="border-bottom"></div></h2>
+    <ul id="event-detail">
+        <li class='events-list'><img src="images/icons8-twitch-50.png" class="event-img" alt=""><h2 class="list-info">Streaming</h2><p class="match-info">Watch all matches live exclusively on twitch</p></li>
+        <li class='events-list'><span class="material-symbols-outlined" style="font-size:48px;" >
+            schedule
+            </span><h2 class="list-info">Timmings</h2><p class="match-info">Starts at 08:00am Utc-1</p></li>
+        <li class='events-list'><span class="material-symbols-outlined" style="font-size:48px;">
+            local_fire_department
+            </span><h2 class="list-info">Rivalary Match</h2><p class="match-info">Previously unsettled matches to be played</p></li>
+        <li class='events-list'><img src="images/vote.png" alt="" class="event-img"><h2 class="list-info">Vote</h2><p class="match-info">Reach us and vote for your favourite Hero arcana.</p></li>
+        <li class='events-list'><img src="images/contact.png" alt=""><h2 class="list-info">Contact</h2><p class="match-info">Contact us for more details</p></li>
+    </ul>
+    <div class="join-us"><h2 class="join-community">Become the part of this year mega event</h2></div>
+    <div class="event-link-container"><a class="event-link">See the Whole Event</a></div></section>
+    <div id="dynamic-speakers"></div>
+    <div class="partner-container-desktop">
+        <h2 class="partner-heading">Partner</h2>
+        <div class="partner-border"></div>
+        <div class="partners-logo-container">
+        <img src="images/water-drop.png" alt=""class='partner-logo1'>
+    <img src="images/google.png" alt=""class='partner-logo1'>
+    <img src="images/asus.png" alt=""class='partner-logo1'><img src="images/nvidia.png" alt=""class='partner-logo1'>
+    <img src="images/energy-drink.png" alt="" class="partner-logo1"></div>
+    </div>
+    <div class="copy-right-container mobile"><img src="images/atom.png" alt=""class='dota2-logo-img2'><h2 class="copy-right-heading">Dota2 ti 2022 all rights reserved.</h2></div>
```

Comment:
- [ ] Kindly add a description of the image in all of the `alt` attributes.

Reply:
fixed